tests/data/journals/long_title.journal

```diff
@@ -0,0 +1,9 @@
+
+[2022-07-02 00:10] Magna fermentum iaculis eu non diam phasellus. Non pulvinar neque laoreet suspendisse interdum consectetur libero id. Scelerisque felis imperdiet proin fermentum leo. Eu ultrices vitae auctor eu augue ut lectus. Bibendum arcu vitae elementum curabitur vitae nunc sed. Tincidunt tortor aliquam nulla facilisi cras fermentum. Malesuada nunc vel risus commodo viverra maecenas accumsan lacus vel. Non sodales neque sodales ut. Enim nulla aliquet porttitor lacus luctus accumsan. Volutpat blandit aliquam etiam erat velit scelerisque in dictum non. Egestas fringilla phasellus faucibus scelerisque. At risus viverra adipiscing at in tellus integer feugiat scelerisque. Eget velit aliquet sagittis id consectetur purus ut. Imperdiet nulla malesuada pellentesque elit eget gravida cum sociis. Lacus vestibulum sed arcu non odio euismod lacinia at. Elit pellentesque habitant morbi tristique. Vestibulum lorem sed risus ultricies. Integer eget aliquet nibh praesent tristique magna sit amet purus. Quisque id diam vel quam elementum pulvinar etiam non quam. Nisi scelerisque eu ultrices vitae auctor eu augue.
+
+Malesuada fames ac turpis egestas integer eget aliquet. Feugiat scelerisque varius morbi enim nunc faucibus a. At lectus urna duis convallis convallis tellus id interdum velit. Convallis convallis tellus id interdum velit laoreet. Odio eu feugiat pretium nibh ipsum consequat. Potenti nullam ac tortor vitae purus faucibus ornare. Ac auctor augue mauris augue neque gravida in. Quam adipiscing vitae proin sagittis nisl rhoncus mattis. Tempor nec feugiat nisl pretium. Scelerisque in dictum non consectetur a erat. Lacus suspendisse faucibus interdum posuere lorem. Ultricies mi quis hendrerit dolor magna. Morbi tincidunt ornare massa eget egestas purus viverra. Magna fermentum iaculis eu non diam phasellus vestibulum lorem sed. Mauris a diam maecenas sed enim ut sem viverra aliquet. Sollicitudin nibh sit amet commodo nulla facilisi nullam vehicula.
+Nec dui nunc mattis enim ut tellus elementum sagittis vitae. Id porta nibh venenatis cras sed felis. Id leo in vitae turpis. Lorem ipsum dolor sit amet consectetur adipiscing elit pellentesque. Dignissim enim sit amet venenatis urna cursus eget nunc scelerisque. Donec ac odio tempor orci dapibus ultrices in iaculis nunc. Velit laoreet id donec ultrices tincidunt arcu non. Egestas diam in arcu cursus euismod quis. Dolor morbi non arcu risus quis. Nisi lacus sed viverra tellus in hac habitasse platea. Mattis enim ut tellus elementum sagittis vitae et leo. Non pulvinar neque laoreet suspendisse interdum. Libero volutpat sed cras ornare arcu dui vivamus.
```

Comment:
Could you re-write the tests to use the same series of `basic_` config files as the others, then use the "When we run" step to create the long entries, rather than adding this file? That way, we can keep our test data down while also ensuring that this works with every type of journal.

Reply:
Thanks for pointing that out, better that way! Resolved it by removing unneeded changes to config files/journals.